recipe/emailpassword/authFlow_test.go

```diff
@@ -3177,3 +3186,89 @@ func TestSignUpAPIWorksWhenInputIsFine(t *testing.T) {
 	assert.Equal(t, "OK", result["status"])
 	assert.Equal(t, "random@gmail.com", result["user"].(map[string]interface{})["email"])
 }
+
```

Comment:
also need to add a test for a field being optional, and the input doesn't have the field at all, and it should be allowed

Reply:
Added